keps/sig-cli/2590-kubectl-subresource/README.md

```diff
@@ -234,7 +238,11 @@ to implement this enhancement.
 
 #### GA
 
-- User feedback gathered for at least 1 cycle.
+Since v1.27 when the feature moved to beta, there have been no reported bugs concerning this feature.
+In fact, it is reassuring to see the community use this feature quite commonly such as in bug reports:
+https://github.com/kubernetes/kubernetes/issues/116311 
+
+Seeing this and given our added unit, integration and e2e tests gives us the confidence to graduate to GA.
 
 
 ### Upgrade / Downgrade Strategy
```

Comment:
Please update all the relevant PRR/version skew sections to consider server side changes like https://github.com/kubernetes/kubernetes/pull/103516